### PR TITLE
merge financial crew

### PR DESCRIPTION
**Pull request by USTAADCOM**

### CODE REVIEW

File: 1.png
Comment (maximiliangarmatsch) [inline 1.png:1]: movable to subfolder?

File: README.md [hunk 1]
```diff
@@ -37,7 +37,23 @@ python -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 ```
+### create .env
+Create two .envs in main project folder and inside financial_crew
 
+In project main folder
+```
+In project main folder
```
Comment (maximiliangarmatsch) [inline README.md:45]: duplicate line

File: README.md [hunk 2]
```diff
@@ -37,7 +37,23 @@ python -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 ```
+### create .env
+Create two .envs in main project folder and inside financial_crew
 
+In project main folder
```
Comment (maximiliangarmatsch) [inline README.md:43]: in ".env" file, right?

File: error_screenshot.png
Comment (maximiliangarmatsch) [inline error_screenshot.png:1]: move to subfolder

File: financial_crew/crew.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+)
+
+@CrewBase
+class FinnaceCrew():
+	"""Finance crew"""
+	agents_config = './config/agents.yaml'
+	tasks_config = './config/tasks.yaml'
+
+	@agent
+	def admin_assistant_managr(self) -> Agent:
```
Comment (maximiliangarmatsch) [inline financial_crew/crew.py:27]: type "managr"

File: financial_crew/run_crew.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+"""
```
Comment (maximiliangarmatsch) [inline financial_crew/run_crew.py:1]: comment not needed, right?

File: src/utils/helper_funtion.py
Comment (maximiliangarmatsch) [inline src/utils/helper_funtion.py:1]: this is going to be a problem, kayode did lots of cleanup and these filesgot split up today.
also, the pyAutoGui stuff really doesnt belong in this repo (but maybe you just did it for testing?)
let's discuss this live together

File: src/utils/pyautogui_gmail.py
Comment (maximiliangarmatsch) [inline src/utils/pyautogui_gmail.py:1]: same

File: README.md (hunk 1 above)
Comment (USTAADCOM) [reply README.md:45]: Resolved

File: financial_crew/crew.py (hunk 3 above)
Comment (USTAADCOM) [reply financial_crew/crew.py:27]: Resolved

File: financial_crew/run_crew.py (hunk 4 above)
Comment (USTAADCOM) [reply financial_crew/run_crew.py:1]: Resolved

File: README.md (hunk 2 above)
Comment (USTAADCOM) [reply README.md:43]: Yes

File: 1.png
Comment (USTAADCOM) [reply 1.png:1]: moved to screenshots

File: error_screenshot.png
Comment (USTAADCOM) [reply error_screenshot.png:1]: moved to screenshots

File: src/utils/helper_funtion.py
Comment (USTAADCOM) [reply src/utils/helper_funtion.py:1]: will discuss in meeting

**Review by maximiliangarmatsch — APPROVED** [review]

**Pull request merged**